OFL.txt

```diff
@@ -1,3 +1,5 @@
+Copyright 20** The Unified Font Repository Project Authors (https://github.com/googlefonts/Unified-Font-Repository)
+
 This Font Software is licensed under the SIL Open Font License, Version 1.1.
 This license is copied below, and is also available with a FAQ at:
 http://scripts.sil.org/OFL
```

Comment:
let's use directly https

Reply:
Thanks! I pushed an update that uses https and adds a second blank line.